tests/test_sunsch.py

```diff
@@ -374,8 +373,8 @@ def test_starttime(readonly_datadir):
         "insert": [{"filename": "foo1.sch", "days": 10}],
     }
     sch = sunsch.process_sch_config(sunschconf)
-    # Dates are rounded down, clock-times are not supported
```

Comment:
can you check the assert in line 368 above, that 00:00 is not included? (unless we want it included?)

Reply:
It will actually be included - updated assert accordingly.

Reply:
And there are no side-effects in existing usage of `sunsch` when "00:00:00" is included?